library/input/input.css

```diff
@@ -65,3 +65,150 @@
 
 
 /* Search */
+
+
+/* Checkbox Starts*/
+/* https://www.w3schools.com/howto/howto_css_custom_checkbox.asp */
```

Comment:
This line should be removed

Reply:
Nope, this is for reference in the future as well.